app/views/settings/_min_width_sort_criteria.html.erb

```diff
@@ -0,0 +1,166 @@
+          </select>
+
+          <label for="<%= column %>_input" style="margin-left: 0;width:180px"><%= column_captions[column] || column.capitalize %></label>
+          <input type="text" id="<%= column %>_input" name="<%= column %>_input" placeholder="min-width" />
+        </div>
+      <% end %>
+    </div>
+  </div>
+</fieldset>
+
+<%= hidden_field_tag "settings[columns_sorting]", nil, id: "columns_sorting_field" %>
+<%= hidden_field_tag "settings[columns_min_width]", nil, id: "columns_min_width_field" %>
+
+<script>
+  function updateHiddenFields() {
+    const selectedOptions = $('#sorting_columns option');
+    let sortingString = '', widthString = '';
+
+    selectedOptions.each(function() {
+      const columnName = $(this).val();
+      const sortOrder = document.getElementById(columnName + '_sort').value;
+      const minWidth = document.getElementById(columnName + '_input').value;
+      if (sortOrder !== "null")
+        sortingString += `${columnName}:${sortOrder.toUpperCase()},`;
+      if (minWidth)
+        widthString += `${columnName}:${minWidth},`;
+    });
+
+    $('input[name="settings[columns_sorting]"]').val(sortingString.slice(0, -1) /* Remove the last comma */);
+    $('input[name="settings[columns_min_width]"]').val(widthString.slice(0, -1) /* Remove the last comma */);
+  }
+
+  $(document).ready(function() {
+    // Parse columns_min_width string
+    const columnsMinWidthString = `<%= columns_min_width %>`;
+    const columnsMinWidth = columnsMinWidthString
+      ? columnsMinWidthString.split(',').reduce((acc, curr) => {
+          const [columnName, width] = curr.split(':');
+          if (columnName && width) acc[columnName.trim()] = width.trim();
+          return acc;
+        }, {})
+      : {};
+
+    // Populate min-width inputs
+    $.each(columnsMinWidth, function(columnName, minWidth) {
+      const inputElement = document.getElementById(columnName + '_input');
+      if (inputElement) 
+        inputElement.value=minWidth;
+    });
+
+    // Parse columns_sorting string
+    const columnsSortingString = `<%= columns_sorting %>`;
+    const columnsSorting = columnsSortingString
+      ? columnsSortingString.split(',').map(criterion => {
+          const [columnName, order] = criterion.split(':');
+          return columnName && order ? { [columnName.trim()]: order.trim().toLowerCase() } : null;
+        }).filter(Boolean)
+      : [];
+
+    // Populate sorting options
+    $.each(columnsSorting, function(index, sorting) {
+      const columnName = Object.keys(sorting)[0];
+      const sortOrder = sorting[columnName];
+      const sortElement = document.getElementById(columnName + '_sort');
+      if (sortElement)
+        sortElement.value=sortOrder;
+    });
+
+    const applyButton = document.querySelector('input[type="submit"][name="commit"]');
+    if (applyButton) 
+    applyButton.addEventListener('click', updateHiddenFields);
+  });
+
+  function moveOptionss(direction) {

```

Comment:
I think you have a typo here? (two s)

Reply:
nu e typo, mai era o functie cu numele asta si nu mergea, pot sa o redenumesc moveColumnPriority

Reply:
Am inteles. Ok s-o redenumesti in `moveColumnPriority`